Makefile

```diff
@@ -462,16 +435,16 @@ package-electron: $(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-
 package-cli: $(BUILD_DIRECTORY)/$(APPLICATION_NAME)-cli-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH)
 
 ifeq ($(TARGET_PLATFORM),darwin)
-electron-installer-app-zip: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).zip
-electron-installer-dmg: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).dmg
+electron-installer-app-zip: $(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-mac.zip
+electron-installer-dmg: $(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION).dmg
```

Comment:
Hmmm, seems weird that they allow changing the build artefact name in Windows but not in other OSes? \*shrug\*

If we're going to be switching to `electron-builder`-recommended names, then it probably makes sense to switch to the 'recommended' filename on Windows too? Although I guess these changed filenames will also impact @craig-mulligan 's code on the https://github.com/resin-io/etcher-homepage (the next time we do a 'proper' release). And I guess changing filenames might make https://github.com/resin-io/etcher-homepage/issues/57 more important too?

Alternatively, we could stick with the download filenames we're already using, and then simply rename the file at the same time we move it to `$(BUILD_OUTPUT_DIRECTORY)`.

I have no strong opinions on which option we go with.